operations/iuf/workflows/install_or_upgrade_additional_products_with_iuf.md

```diff
@@ -15,8 +15,10 @@ recipe release is deployed and all management NCNs and managed compute and appli
 rebooted to new images and CFS configurations. Manual operations are documented for procedures that
 are not currently managed by IUF.
 
-The initial install workflow comprises the following procedures which must be executed in the order
-shown.
+The initial install workflow comprises the following procedures. The diagram visualizes the workflow and
```

Comment:
Replace "visualizes" with "displays" or "shows".  

"Visualizes" indicates the reader would have to read/examine our content and then create a mental image for themselves.